chrome/content/zotero/elements/attachmentAnnotationsBox.js

```diff
@@ -24,49 +24,49 @@
 */
 
 {
-	class AttachmentAnnotationsBox extends XULElementBase {
+	class AttachmentAnnotationsBox extends ItemPaneSectionElementBase {
 		content = MozXULElement.parseXULToFragment(`
 			<collapsible-section data-l10n-id="section-attachments-annotations" data-pane="attachment-annotations">
 				<html:div class="body">
 				</html:div>
 			</collapsible-section>
 		`);
+
+		get tabType() {
+			return this._tabType;
+		}
+
+		set tabType(tabType) {
+			this._tabType = tabType;
+			this._updateHidden();
+		}
```

Comment:
The getter would just be called on `ItemPaneSectionElementBase` without this, no? And then if we need custom setters, seems like we should call `super(tabType)` and then do any extra work in these subclasses.

Reply:
make sense!

Reply:
OK updated to use `super` as much as possible; but seems we cannot overwrite only the `setter` or `getter` and have to always inherit both.